src/javascript/app_2/Modules/Reports/Containers/open-positions.jsx

```diff
@@ -0,0 +1,98 @@
+import { PropTypes as MobxPropTypes }      from 'mobx-react';
+import PropTypes                           from 'prop-types';
+import React                               from 'react';
+import { withRouter }                      from 'react-router-dom';
+import { localize }                        from '_common/localize';
+import DataTable                           from 'App/Components/Elements/DataTable';
+// import { getContractPath }                 from 'App/Components/Routes/helpers';
+import { connect }                         from 'Stores/connect';
+import EmptyTradeHistoryMessage            from '../Components/empty-trade-history-message.jsx';
```

Comment:
nitpick, i think it is more readable if we use absolute path instead of relative.